config/db.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+
+const MONGO_URI = process.env.MONGODB_CONNECTION_STRING;
```

Comment:
I would have this envar named the same as it's variable, so MONGO_URI.

Also some fun things with JS, a better way to write this would be:

`const { MONGO_URI } = process.env;`

It's called object destructuring if you want to look into it (don't)